code/datums/martial/knifeboxing.dm

```diff
@@ -0,0 +1,58 @@
+/datum/martial_art/knifeboxing
+	name = "Knife-boxing"
+
+/datum/martial_art/knifeboxing/disarm_act(mob/living/carbon/human/A, mob/living/carbon/human/D)
+	to_chat(A, span_warning("Can't disarm while knife-boxing!"))
+	return TRUE
+
+/datum/martial_art/knifeboxing/grab_act(mob/living/carbon/human/A, mob/living/carbon/human/D)
+	to_chat(A, span_warning("Can't grab while knife-boxing!"))
+	return TRUE
+
+/datum/martial_art/knifeboxing/harm_act(mob/living/carbon/human/A, mob/living/carbon/human/D)
+
+	A.do_attack_animation(D, ATTACK_EFFECT_PUNCH)
+
+	var/atk_verb = pick("left hook","right hook","straight punch")
+
+	var/damage = rand(8, 12) + A.dna.species.punchdamage
```

Comment:
Damage may be a bit high, that's 13-17, or 15-19 on lizards, for a weapon that can't be disarmed